py/kubeflow/testing/ks_util.py

```diff
@@ -0,0 +1,55 @@
+"""Utilities for working with ksonnet in the tests."""
+
+import filelock
+import logging
+import os
+import re
+import subprocess
+import yaml
+
+from kubeflow.testing import util
+
+def setup_ks_app(app_dir, env, namespace, component, params, ks_cmd=None):
```

Comment:
Will there be a separate PR to migrate other code (e.g. in tf-operator) to call this method?